ai/mcp/src/test/java/com/linecorp/armeria/server/ai/mcp/ArmeriaStatelessIntegrationTests.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2025 LY Corporation
+ *
+ * LY Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.ai.mcp;
+
+import java.time.Duration;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.params.provider.Arguments;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.logging.LoggingService;
+
+import io.modelcontextprotocol.AbstractStatelessIntegrationTests;
+import io.modelcontextprotocol.client.McpClient;
+import io.modelcontextprotocol.client.transport.HttpClientStreamableHttpTransport;
+import io.modelcontextprotocol.client.transport.WebClientStreamableHttpTransport;
+import io.modelcontextprotocol.server.McpServer;
+import io.modelcontextprotocol.server.McpServer.StatelessAsyncSpecification;
+import io.modelcontextprotocol.server.McpServer.StatelessSyncSpecification;
+
+@Timeout(15)
+class ArmeriaStatelessIntegrationTests extends AbstractStatelessIntegrationTests {
+
+    private static final String CUSTOM_MESSAGE_ENDPOINT = "/otherPath/mcp/message";
+
+    private Server httpServer;
+
+    private ArmeriaStatelessServerTransport mcpStreamableServerTransport;
+
+    static Stream<Arguments> clientsForTesting() {
```

Comment:
This isn't used.

Reply:
The method is used as the input of `@MethodSource`.
https://github.com/modelcontextprotocol/java-sdk/blob/8ac7eae47cc8dba4794ab3950e34417d6f6c665b/mcp-test/src/main/java/io/modelcontextprotocol/AbstractStatelessIntegrationTests.java#L53